Yoast/wordpress-seo
# Additional default allowed hosts in webpack.config.js

## Summary

This PR can be summarized in the following changelog entry:

* Since the VVV default configuration has been updated (see [here](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1650)), the local WordPress installations have been renamed `wordpress-one` and `wordpress-two` rather than `wordpress-local`.
* This PR adds these two installs to the default allowed hosts in the webpack config. 

## Test instructions

This PR can be tested by following these steps:

* Start `one.wordpress.test` in your browser (you have to have the local VVV installation called `wordpress-one`).
* In the browser console, find the error message `ERR_CONNECTION_REFUSED`.
* Check out this branch.
* After building and running `yarn start`, the error message should have disappeared. 
* The same steps apply to `two.wordpress.test`.

## UI changes

## Quality assurance
